### ISSUE TITLE
[GitHub Sync]: Obfuscate Personal Access Token input

### ISSUE THREAD

**Issue by six7** [six7/figma-tokens#340]
Context:
```
Would it a possibility to have a password field for the Personal Access Token in the GitHub sync panel?
Reason I ask is that I was demo’ing Figma Tokens outside of my organisation and unfortunately the private key was accidentally displayed. Anyone capturing a screenshot now potentially has access to our design tokens.
```

```
Good point, what if we obfuscated it? And you could toggle that so whenever you need to edit it you could, but nobody would see it otherwise
```

**Comment by pietgk** [comment]
maybe https://github.com/AGWA/git-crypt is an idea?

**Comment by six7** [comment]
Fixed in #491 - will release this week
